src/main/java/edu/kis/powp/jobs2d/events/SelectClearPanelOptionListener.java

```diff
@@ -1,13 +1,18 @@
 package edu.kis.powp.jobs2d.events;
 
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.adapter.AbstractAdapter;
+import edu.kis.powp.jobs2d.features.DriverFeature;
+
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 
-import edu.kis.powp.jobs2d.features.DrawerFeature;
-
 public class SelectClearPanelOptionListener implements ActionListener {
 	@Override
 	public void actionPerformed(ActionEvent e) {
-		DrawerFeature.getDrawerController().clearPanel();
+		Job2dDriver driver = DriverFeature.getDriverManager().getCurrentDriver();
+		if (driver instanceof AbstractAdapter) {
+			((AbstractAdapter) driver).clearPanel();
+		}
```

Comment:
Unnecessary changes. The problem was only in Adapter implementation, now the whole code is inconsistent.